import-service/lambda/importProductsFile.ts

```diff
@@ -0,0 +1,61 @@
+import { PutObjectCommand, S3Client } from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+import { APIGatewayEvent } from "aws-lambda";
+
+export const handler = async (event: APIGatewayEvent) => {
+  try {
+    const filename = event.queryStringParameters?.name;
+
+    if (!filename) {
+      const errorMessage = "File name is required";
+      console.log(errorMessage);
+      return {
+        statusCode: 400,
+        headers: {
+          "Access-Control-Allow-Origin": "*",
+          "Access-Control-Allow-Credentials": true,
+          "Access-Control-Allow-Headers": "Content-Type",
+          "Access-Control-Allow-Methods": "GET",
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ message: errorMessage }),
+      };
+    }
+
+    const client = new S3Client();
+    const command = new PutObjectCommand({
+      Bucket: process.env.BUCKET_NAME,
+      ContentType: "text/csv",
+      Key: `uploaded/${filename}`
+    });
+    const signedUrl = await getSignedUrl(client, command, {
+      expiresIn: 3600,
+      signableHeaders: new Set(['content-type'])
+    });
+
+    return {
+      statusCode: 200,
+      headers: {
+        "Access-Control-Allow-Origin": "*",
+        "Access-Control-Allow-Credentials": true,
+        "Access-Control-Allow-Headers": "Content-Type",
+        "Access-Control-Allow-Methods": "GET",
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ signedUrl }),
```

Comment:
there was a requirement to provide signedUrl in the response body as is (just simple string).
```suggestion
      body: signedUrl,
```

This way you don't need to update API contract on FE side

Task 5.2 (point 6)
- The response from the lambda should be clean Signed URL, as a string.